src/main/java/com/dataloom/organization/Organization.java

```diff
@@ -85,30 +111,44 @@ public String getDescription() {
         return roles;
     }
 
+    @JsonProperty( SerializationConstants.APPS )
+    public Set<UUID> getApps() {
+        return apps;
+    }
+
     @Override public boolean equals( Object o ) {
-        if ( this == o ) { return true; }
-        if ( !( o instanceof Organization ) ) { return false; }
+        if ( this == o )
+            return true;
+        if ( o == null || getClass() != o.getClass() )
+            return false;
 
         Organization that = (Organization) o;
 
-        if ( h != that.h ) { return false; }
-        if ( !securablePrincipal.equals( that.securablePrincipal ) ) { return false; }
-        if ( !autoApprovedEmails.equals( that.autoApprovedEmails ) ) { return false; }
-        if ( !members.equals( that.members ) ) { return false; }
-        return roles.equals( that.roles );
+        if ( h != that.h )
```

Comment:
This checks if the hash is equal and will fail for two identical objects when one has never had it's hash code computed.